### PR TITLE
Fix admin ride deletion and add "Approve", "Reject" buttons

### PR DESCRIPTION
**Pull request by narayanrai97**
- Fixed admin ride delete button
- Added flash message display on deletion
@micronix !

### CODE REVIEW

File: app/views/admin_ride/index.html.erb [hunk 1]
```diff
@@ -24,11 +24,17 @@
...
       <td><%= ride.status %></td>
-      <td><%= link_to 'Show', admin_ride_path(ride) %></td>
       <td><%= link_to 'Edit', edit_admin_ride_path(ride) %></td>
-      <td><%= link_to 'Delete', admin_ride_index_path(ride),
-                      method: :delete,
-                      data: { confirm: 'Are you sure?' } %></td>
+      <% if ride.status == 'approved' %>
+        <td><%= link_to 'Approve', "#", class: "btn btn-sm btn-success disabled"%></td>
+      <% else %>
+        <td><%= button_to 'Approve', approve_admin_ride_path(ride), class: "btn btn-sm btn-success", method: :put%></td>
```
Comment (micronix) [inline app/views/admin_ride/index.html.erb]: change this to a link_to to be consistent with the method used above and below

File: app/controllers/admin_ride_controller.rb [hunk 2]
```diff
@@ -95,11 +95,23 @@ def update
   end
 
   def destroy
```
Comment (micronix) [inline app/controllers/admin_ride_controller.rb]: I think we talked about not being able to delete a ride, to keep the history of what happened.

File: app/views/admin_ride/index.html.erb (hunk 1 above)
Comment (narayanrai97) [reply app/views/admin_ride/index.html.erb]: I changed it to `link_to`. Thanks!

File: app/controllers/admin_ride_controller.rb (hunk 2 above)
Comment (narayanrai97) [reply app/controllers/admin_ride_controller.rb]: I took off the `destroy` action to prevent ride deletion!

File: config/application.rb [hunk 3]
```diff
@@ -17,6 +17,7 @@ class Application < Rails::Application
     # the framework and any gems in your application.
     config.time_zone = 'Eastern Time (US & Canada)'
     config.active_record.default_timezone = :local
+    config.cache_classes == true
```
Comment (micronix) [inline config/application.rb]: what is this?

File: app/models/ride.rb [hunk 4]
```diff
@@ -9,8 +9,8 @@ class Ride < ApplicationRecord
 
   validates :pick_up_time, :reason, :status, presence: true
   validate :pick_up_time_cannot_be_in_the_past
-  validates :status, inclusion: { in: %w(requested pending approved scheduled matched canceled picking-up dropping-off completed),
-  message: "%{value} is not a valid status" }
+  # validates :status, inclusion: { in: %w(requested pending approved scheduled matched canceled picking-up dropping-off completed),
```
Comment (micronix) [inline app/models/ride.rb]: why remove the validation?

File: config/application.rb (hunk 3 above)
Comment (narayanrai97) [reply config/application.rb]: I took it off!

File: app/models/ride.rb (hunk 4 above)
Comment (narayanrai97) [reply app/models/ride.rb]: It was an accidental change. I've put it back in place!

**Pull request merged**